packages/cli/src/controllers/project.controller.ts

```diff
@@ -44,14 +44,8 @@ export class ProjectController {
 		const relations = await this.projectsService.getProjectRelationsForUser(req.user);
 
 		return relations.map((pr) => {
-			let name = pr.project.name;
-			// Only personal projects don't have a name and the only
-			// personal project a user should be linked to is their own
-			if (!name) {
-				// TODO: confirm name with product
-				name = 'My n8n';
-			}
 			return {
+				name: `Unclaimed Personal Project (${pr.projectId})`,
```

Comment:
One consideration is that we might want to have all projects with name, including for invited but not signed up user - perhaps use just the email. But this is something to discuss later.